Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -780,7 +798,23 @@ BoardInit (
         DEBUG ((DEBUG_WARN, "SCI device has boot issue\n"));
       }
     }
-    ClearSmbusStatus ();
```

Comment:
why remove this one?  It isn't mentioned in the commit message.

Reply:
The ClearSmbuStatus was designed to clear GPE0_STS_127_96.SMB_WAK_STS and HSTS.B_SMBUS_IO_SMBALERT_STS for SMBUS (B0:D31:F4). However, 
  1. the HSTS.B_SMBUS_IO_SMBALERT_STS should be detected, handled, and cleared by device driver (or proper SMI handler)
  2. the ClearSmi() will clear GPE0_STS_127_96.SMB_WAK_STS if it is set
As a result, ClearSmbuStatus() is an unwanted implementation.

A new commit is submitted to
  1. remove ClearSmbuStatus() code
  2. enhance commit message

Reply:
If I remember correctly, ClearSmBusStatus was added because there are 2 instances to use Smbus to get data in SBL. The second instance doesn't work because Smbus STS is set.  Maybe we should check if this could be handled in SmbusLib if it happens again.